src/deepstream.io.js

```diff
@@ -308,6 +310,9 @@ Deepstream.prototype._init = function() {
 	this._recordHandler = new RecordHandler( this._options );
 	this._messageDistributor.registerForTopic( C.TOPIC.RECORD, this._recordHandler.handle.bind( this._recordHandler ) );
 
+	this._presenceHandler = new PresenceHandler( this._options, this._connectionEndpoint );
```

Comment:
Lets be cheeky and put this on the `options` object instead to so we can call added/remove methods.